examples/plugins/react-native-get-location-plugin/LocationPlugin.ts

```diff
@@ -0,0 +1,37 @@
+import { Types } from '@amplitude/analytics-react-native';
+import GetLocation, { Location } from 'react-native-get-location';
+
+// https://github.com/douglasjunior/react-native-get-location#install - install steps, usage.
+export default class LocationPlugin implements Types.BeforePlugin {
+  name = 'getLocation';
+  type = Types.PluginType.BEFORE as any;
+  location: Location | undefined;
+
+  async setup(_config: Types.Config): Promise<undefined> {
+    GetLocation.getCurrentPosition({
+      enableHighAccuracy: true,
+      timeout: 30000,
```

Comment:
What's the timeout for?

Reply:
The example is copied from `react-native-get-location` library.
https://github.com/douglasjunior/react-native-get-location#object-locationconfig
The max time (in milliseconds) that you want to wait to receive a location.